x-pack/plugins/cases/server/common/utils.ts

```diff
@@ -4,19 +4,38 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
+import Boom from '@hapi/boom';
```

Comment:
I moved most of the functionality from the `routes/api/utils` to here or `client/utils` depending on where they were used.